src/GitTabExpansion.ps1

```diff
@@ -169,23 +169,23 @@ function script:gitConfigKeys($section, $filter, $defaultOptions = '') {
 }
 
 function script:gitTags($filter, $prefix = '') {
-    git tag |
+    git tag --list "$filter*" |
         Where-Object { $_ -like "$filter*" } |
         ForEach-Object { $prefix + $_ } |
         quoteStringWithSpecialChars
 }
 
 function script:gitFeatures($filter, $command) {
     $featurePrefix = git config --local --get "gitflow.prefix.$command"
-    $branches = @(git branch --no-color | ForEach-Object { if ($_ -match "^\*?\s*$featurePrefix(?<ref>.*)") { $matches['ref'] } })
+    $branches = @(git branch --no-color --list "$filter*" | ForEach-Object { if ($_ -match "^\*?\s*$featurePrefix(?<ref>.*)") { $matches['ref'] } })
     $branches |
         Where-Object { $_ -ne '(no branch)' -and $_ -like "$filter*" } |
         ForEach-Object { $featurePrefix + $_ } |
         quoteStringWithSpecialChars
 }
 
 function script:gitRemoteBranches($remote, $ref, $filter, $prefix = '') {
-    git branch --no-color -r |
+    git branch --no-color --remotes --list "$remote/$filter*" |
         Where-Object { $_ -like "  $remote/$filter*" } |
```

Comment:
Can we drop the `Where` if `--list` is filtering?

Reply:
In this case it's probably safe to do since the filtering is very simple, sure!